recipes/rasterstats/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.10.3" %}
+
+package:
+    name: rasterstats
+    version: {{ version }}
+
+source:
+    fn: rasterstats-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/r/rasterstats/rasterstats-{{ version }}.tar.gz
+    sha256: 7a4a79d71fbfb15812ee40081a8fd321039e9b26f389f7ec456c657c3298a818
+
+build:
+    number: 0
+    preserve_egg_dir: True
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - shapely
+        - numpy >=1.9
+        - rasterio >=0.27
+        - cligj >=0.4
+        - fiona
+        - gdal 1.11.4
```

Comment:
Our `gdal 2.x` is currently broken :unamused: 
I will remove this `gdal` pin and dependency  her as soon as we fix it. (@perrygeo does `rasterstats` needs any special feature from `gdal 2`?)